test/SequenceEntity/SequenceEntity.js

```diff
@@ -59,6 +59,23 @@ describe('SequenceEntity', () => {
 			}, 50);
 		});
 
+		it('Check completion', done => {
+			const sirenSequenceRoot =  window.D2L.Hypermedia.Siren.Parse(sequenceRootMultipleTopLevel);
```

Comment:
Nitpick: I think that data should be private to the test, if somebody will change object `sequenceRootMultipleTopLevel`  in `test/SequenceEntity/data/sequenceRoot.js` it may break the test

Reply:
I broke it out into its' own folder by the `json` is very long and I didn't want to clutter the test file.